### PR TITLE
10185 add module task_id_display

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by duanyp1991** [comment]
LGTM

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico  @duanyp1991 @siyuancn

File: task_id_display/project_issue_view.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        <record model="ir.ui.view" id="issue_id_display_kanban_inherit">
```
Comment (siyuancn) [inline task_id_display/project_issue_view.xml]: id before model
id="project_issue_kanban_view"

File: task_id_display/project_task_view.xml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        <record model="ir.ui.view" id="task_id_display_tree_inherit">
```
Comment (siyuancn) [inline task_id_display/project_task_view.xml]: id before model,
id follows the ref.

File: task_id_display/project_task_view.xml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+            <field name="name">project_task.task_id_display_form_view</field>
+            <field name="model">project.task</field>
+            <field name="inherit_id" ref="project.view_task_form2"/>
+            <field name="arch" type="xml">
+                <field name="name" position="before">
+                    <field name="id" />
+                </field>  
+            </field>
+        </record>
+        <record model="ir.ui.view" id="task_id_display_kanban_inherit">
```
Comment (siyuancn) [inline task_id_display/project_task_view.xml]: id before model,
id follows the ref.

File: task_id_display/project_task_view.xml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+            <field name="name">project_task.task_id_display_kanban_view</field>
+            <field name="model">project.task</field>
+            <field name="inherit_id" ref="project.view_task_kanban"/>
+            <field name="arch" type="xml">
+                <field name="name" position="before">
+                    <field name="id" />
+                </field>  
+            </field>
+        </record>
+        <record model="ir.ui.view" id="task_id_display_search_inherit">
```
Comment (siyuancn) [inline task_id_display/project_task_view.xml]: id before model,
id follows the ref.

File: task_id_display/README.rst [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+===============
+
+This module extends the functionality of project and allow you to see the task ID from the task views.
+
+Installation
+============
+
+To install this module, you need to:
+
+ * have basic modules installed (project, project_issue)
```
Comment (siyuancn) [inline task_id_display/README.rst]: project_issue is inherited from project.
I thinks we don't need install project here.

**Comment by elicoidal** [comment]
name should be project_task_id

**Comment by Rona111** [comment]
cc @elicoidal @siyuancn @duanyp1991 @seb-elico

**Comment by elicoidal** [comment]
LGTM
BTW: where is the rest of the code? you should have a complementary module

**Comment by siyuancn** [comment]
LGTM

**Comment by Rona111** [comment]
@elicoidal  where is the rest of the code? This module is a part of elico_project (PR ID:31)

**Comment by elicoidal** [comment]
I mean this module contained before more functions that I asked to remove. This features should be placed in an additional module

**Pull request merged**